### ISSUE TITLE
Kubernetes deployment keeps dying

### ISSUE THREAD

**Issue by nunodsfernandes** [polonel/trudesk#596]
**Is this a BUG REPORT or FEATURE REQUEST?**:

- [ x ] BUG 
- [ ] FEATURE

**What happened**:
Kubernetes deployment keeps "dying".
I deployed it on AWS EKS, using the files under `https://github.com/polonel/trudesk/tree/master/kubernetes`.  
After 1 or 2 days of working well, the deployment dies, and it seems to be related to DB.  
Accessing the Trudesk address (Route53 DNS) doesn't show the login page, but the install page again.  
No matter what i insert on the install page to create another admin, it always fails:

![image](https://user-images.githubusercontent.com/75372015/222929060-b86ff8f2-8b97-4515-8429-612a30f7ccaa.png)

Error field:
`{"success":false,"error":{"index":0,"code":11000,"keyPattern":{"name":1},"keyValue":{"name":"gen:version"}}}`

**What did you expect to happen**:

**How to reproduce it (as minimally and precisely as possible)**:
No idea.. Deploy EKS and wait?

**Anything else we need to know?**:
I also restore a DB backup from a VM that i was running.  
That loads perfectly, and everything works, until it dies.

I do not deploy Elasticsearch, because if i do, it remains on CrashLoopBackoff and doesn't load anyway..

**Environment**:
- Trudesk Version:
- OS (e.g. from /etc/os-release):
- Node.JS Version:
- MongoDB Version: 
- Is this hosted on cloud.trudesk.io: NO

Environment based on values found on the Kubernetes deployment, i do not change anything.

**Comment by nunodsfernandes** [comment]
Well, it's been 1 month...

Is Trudesk dead?

**Comment by polonel** [comment]
No. I do not have a k8s cluster so I'm not able to test. The k8s configuration was done by a community member last year. 

With that note, the Install page only shows if it is unable to connect to Mongo during startup to check for a configuration flag. If it cannot connect or part of the ConfigMap or env vars are missing it defaults to the install page. 

The error you're getting is it trying to initialize a DB that has already been initialized.

**Comment by nunodsfernandes** [comment]
Hi Chris,

Yeah, I also got the same impression, however, i cannot, for the life of me, find out what is causing it to try to initialize the DB.
Pod remains healthy, so... It's weird. I've looked at Mongo connection, but also no luck on figuring it out.
Everything seems to be OK.

I'm fine with digging and troubleshooting, but honestly, no idea on what can be causing it.
Thing is that i was moving everything to K8s to save some costs on AWS, and this is the only tool i have that still has to run on VM :/
